src/cow-react/modules/limitOrders/pure/OrderStatusBox/index.tsx

```diff
@@ -48,3 +55,31 @@ export const OrderStatusBox = styled.div<{ status: OrderStatus; cancelling: bool
     border-radius: ${({ withWarning }) => (withWarning ? '9px 0 0 9px' : '9px')};
   }
 `
+
+export type OrderStatusBoxProps = { order: ParsedOrder; widthAuto?: boolean }
+
+export function OrderStatusBox({ order, widthAuto }: OrderStatusBoxProps) {
+  return (
+    <Wrapper
+      style={widthAuto ? { width: 'auto' } : {}}
+      cancelling={!!order.isCancelling}
+      partiallyFilled={order.partiallyFilled}
```

Comment:
why cancelling needa the !! and not partiallyFilled?

Reply:
Was already there but will fix it